blueocean-dashboard/src/main/js/components/RunDetails.jsx

```diff
@@ -57,11 +57,14 @@ function getTestSummaryUrl(runDetails) {
 class RunDetails extends Component {
     constructor(props) {
         super(props);
-        this.state = { isVisible: true };
+        this.state = { isVisible: true, actions: [] };
     }
 
     componentWillMount() {
         this._fetchRun(this.props);
+        Extensions.store.getExtensions(['jenkins.run.actions'], Extensions.Utils.sortByOrdinal, (actions = []) => {
```

Comment:
Collect all extensions of the `jenkins.run.actions` extension point and store in the `RunDetails` component's state.